src/RestControllers/AuthorizationController.php

```diff
@@ -378,7 +380,11 @@ public function newClientSave($clientId, $info): bool
 
 
         try {
-            $sql = "INSERT INTO `oauth_clients` (`client_id`, `client_role`, `client_name`, `client_secret`, `registration_token`, `registration_uri_path`, `register_date`, `revoke_date`, `contacts`, `redirect_uri`, `grant_types`, `scope`, `user_id`, `site_id`, `is_confidential`, `logout_redirect_uris`, `jwks_uri`, `jwks`, `initiate_login_uri`, `endorsements`, `policy_uri`, `tos_uri`) VALUES (?, ?, ?, ?, ?, ?, NOW(), NULL, ?, ?, 'authorization_code', ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
+            $sql = "INSERT INTO `oauth_clients` (`client_id`, `client_role`, `client_name`, `client_secret`,"
+            . " `registration_token`, `registration_uri_path`, `register_date`, `revoke_date`, `contacts`, "
+            . "`redirect_uri`, `grant_types`, `scope`, `user_id`, `site_id`, `is_confidential`, `logout_redirect_uris`,"
+            . "`jwks_uri`, `jwks`, `initiate_login_uri`, `endorsements`, `policy_uri`, `tos_uri`) VALUES (?, ?, ?, ?, "
+            . "?, ?, ?, NOW(), NULL, ?, 'authorization_code', ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)";
```

Comment:
@adunsulag , Something doesn't look right here (more placemakers to left of NOW() in new string). Is this supposed to be changed at all?

Reply:
Gonna put it back into a one liner for a commit to see if any differences.

Reply:
was weird, no changes except for a shifted placemarker (guessing had to do with adding something here and then removing it from wrong place in the placemarkes). I reverted the change.

Reply:
I'm not sure what is going on, that isn't what my file looks like and it passes the tests.  It's like their some weird merge stuff going on between my local branch and what shows up in the pull request.

Reply:
the joys of git :)

Reply:
Looks like on the original commit that an extra placemaker was added to left of NOW() prob when breaking up the line that was offset by adding the new variable, so didn't break anything. That's why we (probably me) should avoid making these megasize one liner queries :)